Testing/Tests/graveyard_removal_test.py

```diff
@@ -0,0 +1,57 @@
+"""
+This class ensures that GraveyardRemoval.py correctly removes the graveyard from an h5m file.
+"""
+
+import os
+import pymoab
+import sys
+
+sys.path.insert(1,"../../PythonTool")
+
+from GraveyardRemoval import get_sets_by_category, locate_graveyard, format_file_name
+
+
+"Initialize a PyMOAB core instance and load in the h5m file."
+mb = pymoab.core.Core()
+mb.load_file("../SampleData/donut.h5m")
+
+
+def test_get_sets_by_category():
+	# Ensure this function returns the correct number of entities with the specified tag.
+	group_categories = get_sets_by_category(mb, "Group")
+	assert len(group_categories) == 5
+
+
+def test_locate_graveyard():
+	# Ensure this function returns the correct list of entity sets without the graveyard volume.
+    groups_to_write, graveyard_sets = locate_graveyard(mb)
+    assert groups_to_write == [12682136550675318125, 12682136550675318126,
+                               12682136550675318128, 12682136550675318129]
+
+
+def test_default_format_file_name():
+	# Ensure this function returns the correct default output file name.
+	output_name = format_file_name('donut.h5m')
```

Comment:
maybe store `donut.h5m` file name in a variable as you use it in most of the tests